shared/vendor/css/videojs/video-js.css

```diff
@@ -236,7 +236,7 @@
   box-sizing: border-box;
   color: #fff;
   background-color: #fff;
-  position: relative;
+  position: unset;
```

Comment:
Can we override this in the OAE CSS rather than change the vendor code? This will ensure that updating video.js will more likely not have any regressions.